plugins/node/instrumentation-dataloader/package.json

```diff
@@ -51,7 +51,7 @@
     "@opentelemetry/sdk-trace-base": "^1.6.0",
     "@opentelemetry/sdk-trace-node": "^1.6.0",
     "@types/mocha": "7.0.2",
-    "@types/node": "16.11.21",
+    "@types/node": "^14",
```

Comment:
This repo has a renovate config which pins devDependencies so this will create a followup PR.
Please pin it to a reasonable version. Not sure if there is a consistent version in other instrumentation. if yes reuse, if no choose your favorite :o)